utils/ruleset-whitelist.csv

```diff
@@ -1,4 +1,5 @@
 hash,auto-pass coverage test,auto-pass fetch test,filename
+030dda050df8728653c6965bc82a9efbb05f80f04e3f3379a7ef3e114fb94e33,0,1,WyndhamHotels.com.xml
```

Comment:
Can we just retry without whitelisting this ruleset? It may be a transient failure.